packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -880,7 +852,11 @@ private int getActivityHashCode() {
     if (registrar != null && registrar.activity() != null) {
       return registrar.activity().hashCode();
     } else {
-      return activityHashCode;
+      // This method should only be invoked in cases where registrar != null, and only at lifecycle
```

Comment:
This exception can actually throw when some lifecycle event happened when the activity is in the background as mentioned in https://github.com/flutter/plugins/pull/3206/files#r512392320
It is a regression caused by https://github.com/flutter/plugins/pull/2488. We should fix it by removing the `getActivityHashCode()` and use a cached hashCode value like how it was.
I created an issue to track this fix: https://github.com/flutter/flutter/issues/69128

@math1man Do you mind reverting it back to return -1 here and add a TODO to fix the regression?
Something like:
```
// TODO(cyanglaz): Remove the `getActivityHashCode()` and use a cached hashCode when creating the web view for V1 embedding.
// https://github.com/flutter/flutter/issues/69128
return activityHashCode;
```

Reply:
Done, thanks for creating that.